wp-updates-notifier.php

```diff
@@ -196,12 +205,8 @@ public static function disable_cron() {
 		 *
 		 * @return array $links
 		 */
-		public static function plugin_action_links( $links, $file ) {
-			static $this_plugin;
-			if ( !$this_plugin ) {
-				$this_plugin = plugin_basename( __FILE__ );
-			}
-			if ( $file == $this_plugin ) {
+		public function plugin_action_links( $links, $file ) {
+			if ( $file == plugin_basename( __FILE__ ) ) {
```

Comment:
Did a simplification on this to avoid the anti-pattern with global variable.

`plugin_basename()` is fast enough to not require caching of any sort.
If caching would be needed a (static) property would be much better.

The issue being possible conflicts with other code using the same variable and a rather minor concern that `$this_plugin` can be overloaded with executable code.